nafas/__main__.py

```diff
@@ -28,13 +32,20 @@ def main():
             if silent_flag:
                 tprint("Silent Mode")
             description_print()
-            _ = input("Press any key to continue.")
-            clear_screen()
```

Comment:
it's not super important but I wanted to know if you accidentally removed the `clear_screen()`?

Reply:
No, it wasn't removed by accident; it was redundant.